sshd-core/src/main/java/org/apache/sshd/server/forward/TcpipServerChannel.java

```diff
@@ -260,6 +260,20 @@ protected void handleChannelOpenFailure(OpenFuture f, Throwable problem) {
 
     @Override
     public CloseFuture close(boolean immediately) {
+
+        if (!immediately && (out != null)) {
+            try {
+                if (log.isDebugEnabled()) {
+                    log.debug("Closing ChannelOutputStream: {}", TcpipServerChannel.this);
+                }
+                out.close();
```

Comment:
No completely sure this is the correct way (`TcpipServerChannel.close(false)` is also called in case of `handleWriteDataFailure` and `ioHandler.exceptionCaught`) but it seems to be the only  way for the `SSH_MSG_CHANNEL_EOF` message to be send back to the ssh client